dice-engine/README.md

```diff
@@ -33,17 +33,25 @@ problem in Clojure.
 
 **Dice-roll notation:** `NdX`
 
+**Die map representation**
+
+```clojure
+(def die {:value           1
+          :discarded       false
+          :number_of_faces 3
```

Comment:
Do you need to call it `number_of_faces`? Just `faces` would make sense no? Since a die has a certain number of faces is implied?

Reply:
Just faces make sense, I'll rename it